settings.json.template

```diff
@@ -409,6 +409,25 @@
   "indentationOnNewLine": false,
   */
 
+  /*
+   * Import Rate limiting, File size and If a session is required to import
+   * 
+   * importRateLimiting defaults at 10 requests per IP per 90 seconds.  After this it's rejected, value is in mS.  See express-rate-limit for more options
+   * requireAuthorSessionToImport defaults to true meaning that an author must be on the pad to import.
+   * importMaxFileSize defaults to 50Mb meaning that a single imported file max size is 50Mb.  File size is specified in bytes.
+   * 
+   * Uncomment to customize.
+   */
+
+  /*
+  "importRateLimiting": {
```

Comment:
Looking at the code, the rate limiting is applied (correctly, IMHO), on both export and import routes:

https://github.com/ether/etherpad-lite/blob/bb409df1d82e78aba7499e12ff6ffd7db011ed60/src/node/hooks/express/importexport.js#L12

https://github.com/ether/etherpad-lite/blob/bb409df1d82e78aba7499e12ff6ffd7db011ed60/src/node/hooks/express/importexport.js#L12

The setting name should reflect it.

I'd use `importExportRateLimiting`, unless we find something less verbose.

Reply:
Ack feel free to change setting name, that's a good shout!

Reply:
Changed the name of the config section: `importRateLimiting` -> `importExportRateLimiting`.

Reworded the description.